Source/EasyNetQ/RabbitAdvancedBus.cs

```diff
@@ -90,23 +90,25 @@ AdvancedBusEventHandlers advancedBusEventHandlers
     }
 
     /// <inheritdoc />
-    [Obsolete("IsConnected is deprecated because it is misleading. Please use GetConnectionStatus instead")]
     public bool IsConnected =>
-        (from PersistentConnectionType type in Enum.GetValues(typeof(PersistentConnectionType)) select GetConnection(type))
-        .All(connection => connection.Status.State == PersistentConnectionState.Connected);
+        (from PersistentConnectionType type in Enum.GetValues(typeof(PersistentConnectionType)) select GetConnectionStatus(type))
+        .All(status => status.State == PersistentConnectionState.Connected);
 
     /// <inheritdoc />
-    [Obsolete("IsConnected is deprecated because it is misleading. Please use GetConnectionStatus instead")]
     public async Task ConnectAsync(CancellationToken cancellationToken = default)
     {
         foreach (PersistentConnectionType type in Enum.GetValues(typeof(PersistentConnectionType)))
         {
-            try
-            {
-                await EnsureConnectedAsync(type, cancellationToken).ConfigureAwait(false);
-            }
-            catch (AlreadyClosedException)
+            var connectionStatus = GetConnectionStatus(type);
```

Comment:
These changes seem to be not relevant to the initial goal of this PR, please roll them back.